src/main/java/pl/edu/mimuw/chess/Goniec.java

```diff
@@ -0,0 +1,105 @@
+package pl.edu.mimuw.chess;
+
+import pl.edu.mimuw.Figura;
+
+import java.util.ArrayList;
+
+public class Goniec implements Figura {
+  public Kolor kolor;
```

Comment:
to nie moze być publiczne 😞 wtedy każdy moze zmieniac zawartosc tych pol i robi sie balagan, a nie tego chcemy w programowaniu obiektowym